yelp_package/dockerfiles/itest/k8s/wait_paasta_api.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+count=1
+
+host="("
+shift
+cmd="$@"
+
+until [ $(curl -o /dev/null -s -w %{http_code} http://127.0.0.1:${PAASTA_API_PORT}/v1/version) = 200 ]; do
+      >&2 echo "PaaSTA API is unavailable."
+        sleep 1
+        count=$((${count}+1))
+        if [ ${count} -gt 9 ]; then
```

Comment:
I'd recommend running `shellcheck` on this script, which will probably tell you to put double quotes around the URL so that if PAASTA_API_PORT ends up with spaces in it, word splitting doesn't cause the URL to get split into multiple arguments for curl.

You probably also want double quotes around the whole `$(curl ...)` for similar reasons.